core/src/main/java/com/oath/oak/OakBuffer.java

```diff
@@ -12,17 +12,8 @@
  * This buffer may represent either a key or a value for read access.
  * It mimic the standard interface of Java's ByteBuffer, for example, int getInt(int index), char getChar(int index),
  * capacity(), etc.
- * <p>
- * It is attached to the scope of the callback method it were first introduced to the user.
- * The behaviour of this buffer outside their attached scope is undefined.
- * Such callback method might be the application's serializer and comparator, or a lambda function that can
- * read/store/update the data.
- * This access reduces unnecessary copies and deserialization of the underlying data.
- * In their intended context, the user does not need to worry about concurrent accesses and memory management.
- * Using these buffers outside their intended context may yield unpredicted results, e.g., reading non-consistent data
- * and/or irrelevant data.
  */
-public interface OakReadBuffer {
+public interface OakBuffer {
```

Comment:
Didn't we say it is not going to be public?

Reply:
Correct. Will fix.

Reply:
After working on Druid, I realized I need a common interface between all types of buffers.
So I think this should remain public.

Reply:
This is a way for the developer to state: "I don't care what kind of buffer I get (scoped/unscoped/whatever)".

Reply:
Hmm... OK...

Reply:
Then something need to be said about it in README PR#88